### PR TITLE
bengali keyboard

### LINKED ISSUE #157: No Bangla keyboard : there is no bangla keyboard layout on Ubuntu touch.
There is no Bangla keyboard leyout option in Ubuntu Touch to write bangla text.
In Linux Desktop we usually use "Avro" keyboard layout with ibus.
But in the Mobile(ios or android) we use "Ridmik keyboard" to write Bangla text.Ridmik has different type of keyboard layout thats are "avro"(অভ্র), "provat"(প্রভাত) , "jatio"(জাতীয়) etc

Here the link of Avro keyboard webpage : https://www.ridmik.com/

### PR DESCRIPTION
**Pull request by HackerShohag**
Adding a bengali keyboard: fixes #157

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -386,6 +387,15 @@ Depends: ubuntu-keyboard (= ${binary:Version}),
 Description: Ubuntu on-screen keyboard data files - Macedonian
  Data files for the Ubuntu virtual keyboard - Macedonian
 
+Package: ubuntu-keyboard-bengali
+Architecture: any
+Depends: ubuntu-keyboard (= ${binary:Version}),
+         ${misc:Depends},
+         ${shlibs:Depends},
+
```
Comment (Fuseteam) [inline debian/control]: can you remove this line?

File: plugins/bn-avro/qml/qml.pro [hunk 2]
```diff
@@ -0,0 +1,19 @@
+TOP_BUILDDIR = $$OUT_PWD/../../..
+TOP_SRCDIR = $$PWD/../../..
+
+include($${TOP_SRCDIR}/config.pri)
+
+TARGET = dummy
+TEMPLATE = lib
+
+lang_avro.path = "$$UBUNTU_KEYBOARD_LIB_DIR/avro/"
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/qml.pro]: this should be bn-avro aswell

File: plugins/bn-avro/qml/qml.pro [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+lang_avro.files = *.qml *.js
+
+INSTALLS += lang_avro
+
+# for QtCreator
+OTHER_FILES += \
+    Keyboard_avro.qml \
+    Keyboard_avro_email.qml \
+    Keyboard_avro_url.qml \
+    Keyboard_avro_url_search.qml
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/qml.pro]: these files shoul be called bn-avro as well

File: plugins/bn-avro/qml/Keyboard_avro_emaill.qml [hunk 4]
```diff
@@ -0,0 +1,93 @@
+/*
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/Keyboard_avro_emaill.qml]: this filename has a spelling error as well it should be "email" not "emaill" (in addition to it needing to be bn-avro)

File: plugins/bn-avro/qml/qml.pro (hunk 2 above)
Comment (HackerShohag) [reply plugins/bn-avro/qml/qml.pro]: @Fuseteam, Do I need `lang_avro` to be changed `lang_bn-avro` too?

File: plugins/bn-avro/qml/keys/ABackspaceKey.qml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Canonical Ltd.
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/keys/ABackspaceKey.qml]: please do not remove this line, while adding your copyright line is recommended it is also recommended to append not replace

File: plugins/bn-avro/qml/keys/ACharKey.qml [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Canonical Ltd.
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/keys/ACharKey.qml]: here too

File: plugins/bn-avro/qml/keys/key_constants.js [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Canonical Ltd.
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/keys/key_constants.js]: and here

File: plugins/bn-avro/qml/keys/languages.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Canonical Ltd.
```
Comment (Fuseteam) [inline plugins/bn-avro/qml/keys/languages.js]: and here

File: plugins/bn-probhat/qml/Keyboard_bn-probhat.qml [hunk 9]
```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 Abdullah AL Shohag
```
Comment (Fuseteam) [inline plugins/bn-probhat/qml/Keyboard_bn-probhat.qml:3]: i assume there was one here originally the correct format should be 
```
Copyright 2016 Canonical Lt.
Copyright 2021 Abdullah AL Shohagd

File: plugins/bn-probhat/qml/Keyboard_bn-probhat_email.qml [hunk 10]
```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Abdullah AL Shohag
```
Comment (Fuseteam) [inline plugins/bn-probhat/qml/Keyboard_bn-probhat_email.qml:3]: i assume here too

File: plugins/bn-probhat/qml/Keyboard_bn-probhat_url.qml [hunk 11]
```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2021 Abdullah AL Shohag
```
Comment (Fuseteam) [inline plugins/bn-probhat/qml/Keyboard_bn-probhat_url.qml:3]: and here too probably

File: plugins/bn-probhat/qml/Keyboard_bn-probhat_url_search.qml [hunk 12]
```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2021 Abdullah AL Shohag
```
Comment (Fuseteam) [inline plugins/bn-probhat/qml/Keyboard_bn-probhat_url_search.qml:3]: and here

**Comment by peat-psuwit** [comment]
Hello,

Thank you for contributing to UBports. As part of project renaming and the effort to port Ubuntu Touch stack to Ubuntu 20.04, we're incrementally migrating repositories to GitLab.

Your PR is now migrated to:
https://gitlab.com/ubports/core/lomiri-keyboard/-/merge_requests/187
However, you might not be able to push new code into it, as the branch resides in our repository. In case you need to update it, please open a new MR from your (new) fork. I don't have a better solution at the moment.

Sorry for your inconvenience.

**Pull request closed without merging**